docs/gameplay-sessions/gameplay-sessions-reference.mdx

```diff
@@ -0,0 +1,347 @@
+  "path": string, // A string representing the path of this object in the scene hierarchy
+  "scene": string, // A string representing the scene this object is in
+  "tag": string, // A string representing the tag of this object
+  "layer": string, // A string representing the layer of this object
+  "rendererCount": number, // An integer representing the number of renderers on this object
+  "screenSpaceBounds": {
+    "center": {
+      "x": number, // A float representing the x coordinate of the center of the bounds
+      "y": number, // A float representing the y coordinate of the center of the bounds
+      "z": number // A float representing the z coordinate of the center of the bounds
+    },
+    "extents": {
+      "x": number, // A float representing the x coordinate of the extents of the bounds
+      "y": number, // A float representing the y coordinate of the extents of the bounds
+      "z": number // A float representing the z coordinate of the extents of the bounds
+    }
+  },
+  "worldSpaceBounds": {
+    "center": {
+      "x": number, // A float representing the x coordinate of the center of the bounds
+      "y": number, // A float representing the y coordinate of the center of the bounds
+      "z": number // A float representing the z coordinate of the center of the bounds
+    },
+    "extents": {
+      "x": number, // A float representing the x coordinate of the extents of the bounds
+      "y": number, // A float representing the y coordinate of the extents of the bounds
+      "z": number // A float representing the z coordinate of the extents of the bounds
+    }
+  },
+  "position": {
+    "x": number, // A float representing the x coordinate of the position of the object
+    "y": number, // A float representing the y coordinate of the position of the object
+    "z": number // A float representing the z coordinate of the position of the object
+  },
+  "rotation": {
+    "x": number, // A float representing the x component of the rotation of the object
+    "y": number, // A float representing the y component of the rotation of the
+    "z": number, // A float representing the z component of the rotation of the object
+    "w": number // A float representing the w component of the rotation of the object
+  },
+  "rigidbodies": Rigidbody[], // An array of objects representing the rigidbodies on this object
+  "colliders": Collider[], // An array of objects representing the colliders on this object
+  "behaviours": Behaviour[] // An array of objects representing the behaviours on this object
+}
+```
+
+#### KeyboardInput Schema
+
+Within the `inputs` field, an array of objects represents the keyboard input on this tick. Each object has the following
+schema:
+
+```json
+{
+  "startTime": number, // A float representing the time in seconds since the start of the session when this input started
+  "key": string // A string representing the key that was pressed
+}
```

Comment:
As noted in the video, this is not correct (was waiting for another PR, but will fix this soon)